packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-option-multi/gux-option-multi.tsx

```diff
@@ -118,6 +121,33 @@ export class GuxOptionMulti {
     }
   }
 
+  private renderText(): JSX.Element {
+    // The gux-slot-container attribute is used in gux-listbox-multi and gux-dropdown-multi as a selector to get the slotted gux-option-multi text.
+    // This attribute is required because we need to get the slotted text and exclude the screen reader text.
+    if (this.subtext) {
+      return (
+        <div class="gux-option-text">
+          <gux-truncate
+            gux-slot-container
+            ref={el => (this.truncateElement = el)}
+          >
+            <slot />
+          </gux-truncate>
+          <p>{this.subtext}</p>
```

Comment:
Have you a reason for using `<p>` over `<div>` or `<span>` here? Using elements like `<p>` may leads to inconsistencies as they may have different default styles in browsers.

Reply:
No reason, it was just the first one that came to mind. I'll go with span instead.